src/components/Drawer/PoiDrawer/index.tsx

```diff
@@ -22,6 +22,7 @@ import {
 import { PoiStatusType } from "../../../models/poi";
 
 import Drawer from "..";
+import noImage from "../../../assets/images/noImage.svg";
```

Comment:
place it at line 18

Reply:
done, but i'm curious about why? Seems like not according to alphabet order.